### PR TITLE
Simple typings file for TypeScript support

### PR DESCRIPTION
**Pull request by l-Marty-l**
Very primitive and minimalistic type definitions to get started with this package in TypeScript.

### CODE REVIEW

**Comment by JBuijgers** [comment]
Would be very nice to have this. @pedreviljoen this introduces compile time type safety for TypeScript users. Tnx @l-Marty-l , going to copy your typings into my project for now, hope to see this one merged into this project.

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		 * @memberof MenuDrawerProps
+		 */
+		animationTime?: number
+
+		/**
+		 * Value toggling menu overlay or push. When overlay is true, the menu will overlay the background screen. When overlay is false, the menu will push the background screen to the side
+		 * @type bool
+		 * @memberof MenuDrawerProps
+		 */
+		overlay?: bool
```
Comment (JBuijgers) [inline index.d.ts]: The 'bool' type is being used anymore by TypeScript, this should be 'boolean'.

Comment (pedreviljoen) [reply index.d.ts]: Agree, updated to boolean

**Comment by pedreviljoen** [comment]
@JBuijgers 
Thanks will merge this into master branch 👍

File: index.d.ts (hunk 1 above)
Comment (l-Marty-l) [reply index.d.ts]: Thank you for the fix, programmed too much C# on the same day. haha

**Pull request merged**